src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,18 @@
-export const TodoInfo = () => {};
+import { Todo } from '../services';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }: { todo: Todo }) => {
+  if (todo.user === undefined) {
+    return;
+  }
+
+  return (
+    <article
+      data-id={todo.id}
+      className={todo.completed ? 'TodoInfo TodoInfo--completed' : 'TodoInfo'}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+      <UserInfo user={todo.user} />
```

Comment:
let's just add here the check
```suggestion
      {todo.user && <UserInfo user={todo.user} />}
```